src/components/elements/inputs/boolean-list.js

```diff
@@ -39,7 +40,7 @@ export default ({ className, onCommit, property, value }) => {
               checked={isChecked(option.value)}
               disabled={property.disabled}
               onChange={e => onCommit(getValue(e.target.checked, option.value))}
-              name={property.id}
+              name={property.name}
```

Comment:
We use the `id` attribute on the containing div instead. Does it make any sense to have it on each option when it's gonna be the same value for each one of them?